src/bundle/Resources/public/scss/_tables.scss

```diff
@@ -271,3 +271,210 @@
         }
     }
 }
+
+.ibexa-table {
+    font-family: $font-family-sans-serif;
+    font-size: $ibexa-text-font-size;
+    line-height: calculateRem(24px);
+    border-spacing: 0;
+    background-color: $ibexa-color-white;
+
+    &__row:nth-child(odd) {
+        background-color: $ibexa-color-light-300;
+    }
+
+    &__cell:first-child {
+        border-top-left-radius: $ibexa-border-radius;
+        border-bottom-left-radius: $ibexa-border-radius;
+    }
+
+    &__cell:last-child {
+        border-top-right-radius: $ibexa-border-radius;
+        border-bottom-right-radius: $ibexa-border-radius;
+    }
+
+    &__cell.ibexa-table__cell {
+        vertical-align: middle;
+
+        .form-check {
+            display: flex;
+            height: calculateRem(16px);
+            width: calculateRem(16px);
+        }
+    }
+
+    &__header {
+        display: flex;
+        justify-content: space-between;
+        background-color: $ibexa-white;
+        align-items: center;
+        padding: 0 calculateRem(20px);
+        min-height: calculateRem(50px);
+        border-radius: calculateRem(4px) calculateRem(4px) 0 0;
```

Comment:
We have one value "border-radius" as `$ibexa-border-radius` in the variable so maybe we'll add that too?